src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './react-app-env';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+
+  useEffect(() => {
+    const lowerQuery = query.toLowerCase();
+
+    const visibleMovies = moviesFromServer.filter(movie => (
+      movie.title.toLowerCase().includes(lowerQuery)
+        || movie.description.toLowerCase().includes(lowerQuery)
```

Comment:
would be nice to move lower casing logic to helper function, but it's not critical